main/EDTypesMod.F90

```diff
@@ -329,6 +329,13 @@ module EDTypesMod
      integer  ::  ncan(nclmax,maxpft)                           ! number of total   leaf layers for each canopy layer and pft
 
      !RADIATION FLUXES      
+
+     integer  ::  solar_zenith_flag                           ! integer flag specifying daylight (based on zenith angle)
+     real(r8) ::  solar_zenith_angle                          ! solar zenith angle (radians)
```

Comment:
why are solar_zenith_flag and solar_zenith_angle defined as patch variables?  Couldn't that info be held at the site level?

Reply:
it could.  I was just following the precedent of the flag that is passed in from the HLM.